src/mobot_client/payments/payments.py

```diff
@@ -41,7 +42,6 @@ def __init__(
         self.signal = signal
         self.messenger = messenger
         self.logger = logging.getLogger("MOBot.Payments")
-        self._transaction_lock = threading.Lock()
```

Comment:
Nit: The `threading` import can be removed.

Reply:
Removed! Thanks!